### ISSUE TITLE
Can the use of purpose in authorisation request be considered stable?

### ISSUE THREAD

**Issue by MarkCornall** [camaraproject/IdentityAndConsentManagement#80]
Hi,
Can this section on applying purpose in authorisation be considered a stable position?
 https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-API-access-and-user-consent.md#applying-purpose-concept-in-the-authorization-request 

It would not restrict change but A number of markets need clarity to move on with deployments for full list of purposes that may be available so we can be consistent in the knowledge that a document that is not finished is at least stable in parts.

BR

Mark

**Comment by jpengar** [comment]
@MarkCornall Yes Mark, that section of `CAMARA-API-access-and-user-consent.md` document is the result of the short term agreement reached on Issue #32 and must be considered the current stable position on what refers to purpose declaration in the authorization process.

Please see: https://github.com/camaraproject/IdentityAndConsentManagement/issues/32#issuecomment-1771155815 and previous comments. ...and the minutes of the last working group calls ([04/10](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/MeetingMinutes/MOM-2023-10-04.md) and [18/10](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/MeetingMinutes/MOM-2023-10-18.md)) for further details.

CC @bigludo7 @shilpa-padgaonkar @Elisabeth-Ericsson
